app/models/user.rb

```diff
@@ -0,0 +1,85 @@
+class User < ApplicationRecord
+  attr_accessor :remember_token, :activation_token, :reset_token
+  before_save   :downcase_email
+  before_create :create_activation_digest
+  validates :name,  presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
Try to organise your code better. Move constants on top and group attr_accesors, callbacks, validations, relationships etc together, leave some space between the groups.